### DISCUSSION TITLE
Which package manager are you using? (win/mac/linux)

### DISCUSSION THREAD

**Discussion by Trouffman** [DistroAV/DistroAV#1035] [Ideas]
In order to simplify adoption for DistroAV and avoid supply chain attack or inconsistencies. It is important to understand which package manager is used primarily on which OS.

OS : Windows / Mac OS / Linux(flavor) - ubuntu/debian/fedora etc / Other)
Package manager you use : Distro repo / winget / brew / chocolatey ....

This will help decide if we should also control the release part.

I use :
- Winget on Windows 10-11
- Custom Script for Windows Server
- Brew on Mac OS

===============

Trying to identify everywhere DistroAV & NDI Libs are available

# Official DistroAV
## Windows 

[Winget](https://winget.ragerworks.com/package/DistroAV.DistroAV) - [Github](https://github.com/microsoft/winget-pkgs/tree/master/manifests/d/DistroAV/DistroAV)
![WinGet Package Version](https://img.shields.io/winget/v/DistroAV.DistroAV)
```
winget install distroav.distroav
```
 
## MacOS
[Brew](https://formulae.brew.sh/cask/distroav) - [Github](https://github.com/Homebrew/homebrew-cask/blob/master/Casks/d/distroav.rb)
![Homebrew Cask Version](https://img.shields.io/homebrew/cask/v/distroav?link=https%3A%2F%2Fformulae.brew.sh%2Fcask%2Fdistroav)
```
brew install --cask distroav
```

## Linux :

[Flatpak](https://github.com/flathub/com.obsproject.Studio.Plugin.DistroAV)
![Flathub Version](https://img.shields.io/flathub/v/com.obsproject.Studio.Plugin.DistroAV)
```
flatpak install com.obsproject.Studio com.obsproject.Studio.Plugin.DistroAV
obs --system-talk-name=org.freedesktop.Avahi
```

# UnOfficial DistroAV
[Packages for DistroAV identified on Repology](https://repology.org/project/distroav/versions)
[![Packaging status](https://repology.org/badge/vertical-allrepos/distroav.svg)](https://repology.org/project/distroav/versions)

## Windows
[Chocolatey](https://community.chocolatey.org/packages/distroav) - [Maintaainer Process missing]()
![Chocolatey Version](https://img.shields.io/chocolatey/v/distroav)
```
choco install distroav
```

## Linux
- [Nobara Linux](https://wiki.nobaraproject.org/en/modifications/packages) - [Github](https://github.com/Nobara-Project/rpm-sources/tree/42/media/obs-studio-plugin-distroav%20-%20needs%20obs-studio)

- [Arch Linux](https://aur.archlinux.org/packages/distroav) ![AUR Version](https://img.shields.io/aur/version/distroav) 

# UnOfficial NDI (Runtime / Lib / SDK / Tools)
[![Packaging status](https://repology.org/badge/vertical-allrepos/ndi-sdk.svg)](https://repology.org/project/ndi-sdk/versions)

## Windows 
- [NDI Runtime - Winget](https://winget.ragerworks.com/package/NDI.NDIRuntime) - [Github](https://github.com/microsoft/winget-pkgs/tree/master/manifests/n/NDI/NDIRuntime)
![WinGet Package Version](https://img.shields.io/winget/v/NDI.NDIRuntime)
```
winget install NDI.NDIRuntime
```

- [NDI Tools - Winget](https://winget.ragerworks.com/package/NDI.NDITools) - [Github](https://github.com/microsoft/winget-pkgs/tree/master/manifests/n/NDI/NDITools)
![WinGet Package Version](https://img.shields.io/winget/v/NDI.NDITools)
```
winget install NDI.NDITools
```

## Mac
[Brew - NDI Runtime](https://formulae.brew.sh/cask/libndi) - [Github](https://github.com/Homebrew/homebrew-cask/blob/master/Casks/l/libndi.rb)
![Homebrew Cask Version](https://img.shields.io/homebrew/cask/v/libndi)
```
brew install --cask libndi
```

[Brew - NDI Tools](https://formulae.brew.sh/cask/ndi-tools) - [Github](https://github.com/Homebrew/homebrew-cask/blob/master/Casks/n/ndi-tools.rb)
![Homebrew Cask Version](https://img.shields.io/homebrew/cask/v/ndi-tools)
```
brew install --cask ndi-tools
```

## Linux
[NDI SDK - Nobara Linux](https://wiki.nobaraproject.org/en/modifications/packages) - [Github](https://github.com/Nobara-Project/rpm-sources/tree/42/media/ndi-sdk)

NDI SDK - RPM Fusion - [Github](https://github.com/rpmfusion/ndi-sdk)

[Arch Linux - NDI SDK](https://aur.archlinux.org/packages/ndi-sdk) ![AUR Version](https://img.shields.io/aur/version/ndi-sdk?label=AUR%20-%20NDI%20SDK%20version&link=https%3A%2F%2Faur.archlinux.org%2Fpackages%2Fndi-sdk)

**Comment by MarximusMaximus** [comment]
conda-forge on macos and linux and wsl1/2 if I need something segregated from rest of system, otherwise, pretty much whatever the installer package is from the product's website(s)

if i need to download from command line, curl (or git if a repo)

**Comment by MrHinsh** [comment]
>> Custom Script for Windows Server

Chocolatey is the defacto standard for Windows Server until Winget get their act together.

**Comment by Trouffman** [comment]
Opening Post updated with latest known packages.

**Comment by RayneYoruka** [comment]
Winget/choco depending of what I need.. aside from that apt/RPM as per distro dependent unless flatpak.

**Reply by MrHinsh** [reply]
Winget
